MAINTAINER.rst

```diff
@@ -3,21 +3,11 @@ Maintainer's Documentation
 
 Scripts and utilities for maintainers of this package.
 
-Top-level scripts:
-------------------
-
-    * ``env_setup.py`` - setting up a new virtual environment.
-      Uses poetry so if you like using the virtual environment poetry uses,
-      you should be fine.
-    * ``self_check.py`` - validates the repo is in good shape.
-      Uses `pre-commit`_ but you should use ``self_check.py`` instead in case other
-      things are added and as a standard entrypoint common to many repos.
-
-``bin/`` scripts:
+JGT Tools scripts
```

Comment:
There is still a top-level script to be documented: `env_setup.sh`.
I wonder if we shouldn't call that `bootstrap.sh` instead though.

Reply:
Oooooh, I like that.